release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3379,6 +3379,8 @@ $ oc adm release info 4.16.7 --pullspecs
 
 * Previously, `HostedClusterConfigOperator` resource did not delete the `ImageDigestMirrorSet` (IDMS) object after a user removed the `ImageContentSources` field from the `HostedCluster` object. This caused the IDMS object to remain in the `HostedCluster` object. With this release, `HostedClusterConfigOperator` removes all IDMS resources in the `HostedCluster` object so that this issue no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-36766[*OCPBUGS-36766*])
 
+* Previously, in a cluster that runs {product-title} 4.16 with the Telco RAN DU reference configuration, long duration `cyclictest` or `timerlat` tests could fail with maximum latencies detected above `20` us. This issue occured because the `psi` kernel command line argument was being set to `1` by default when cgroups v2 is enabled. With this release, the issue is fixed by setting `psi=0` in the kernel arguments when enabling cgroups v2. The `cyclictest` latency issue reported in link:https://issues.redhat.com/browse/OCPBUGS-34022[*OCPBUGS-34022*] is now also fixed. (link:https://issues.redhat.com/browse/OCPBUGS-37271[*OCPBUGS-37271*])
```

Comment:
I believe it is supposed to be `crgoup v2` rather than `cgroups v2`. See what calls itself the [authoritative documentation on the design, interface and conventions of cgroup v2](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html)
```suggestion
* Previously, in a cluster that runs {product-title} 4.16 with the Telco RAN DU reference configuration, long duration `cyclictest` or `timerlat` tests could fail with maximum latencies detected above `20` us. This issue occured because the `psi` kernel command line argument was being set to `1` by default when cgroup v2 is enabled. With this release, the issue is fixed by setting `psi=0` in the kernel arguments when enabling cgroup v2. The `cyclictest` latency issue reported in link:https://issues.redhat.com/browse/OCPBUGS-34022[*OCPBUGS-34022*] is now also fixed. (link:https://issues.redhat.com/browse/OCPBUGS-37271[*OCPBUGS-37271*])
```

Reply:
:heavy_check_mark: